scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
@bfredl test/ contains libraries for tests and tests, but both can’t be run on their own (tests are only runnable by busted which is a whole bunch of other lua stuff). Also libraries for tests are named “smth.lua”, tests are named “smth_spec.lua”: easily distinguishable. And all are lua, non-lua things have separate directories.

Neither distinguishable naming nor grouping by “scripts are lua and library for scripts is lua” applies here. Last, but not least: I would not expect directory named scripts/ contain any non-scripts, just like bin/ containing something which is not runnable from the command-line, not even any directory with some stuff; `ls -lA {,/usr{,/local}}/bin/*(ND-^*)` shows that Gentoo maintainers agree with me here: found only .keep and a bunch of broken symlinks.